test/deadline_submitter_for_houdini/unit/test_assets.py

```diff
@@ -13,17 +14,38 @@ def test_get_scene_asset_references():
     )
     node = hou.node
     hou.node.type().name.return_value = "deadline-cloud"
-    parm = hou.Parm
+    temp_parm = hou.Parm
     hou.Parm.node.return_value = node
     hou.Parm.name.return_value = "shadowmap_file"
     hou.node.type().nameWithCategory.return_value = "Driver/ifd"
     hou.hipFile.path.return_value = "/some/path/test.hip"
     hou.node.parm().eval.return_value = "/tmp/foo.$F.exr"
+
+    dir_parm = mock.Mock()
+    dir_parm.node.return_value = None
+    dir_parm.evalAsString.return_value = "/path/assets/"
+
+    file_parm = mock.Mock()
+    file_parm.node.return_value = None
+    file_parm.evalAsString.return_value = "/path/asset.png"
+
     hou.fileReferences.return_value = (
-        (None, "$HIP/houdini19.5/otls/Deadline-Cloud.hda"),
-        (parm, "temp:$OS.rat"),
+        (dir_parm, "$HIP/houdini19.5/"),
+        (file_parm, "$HIP/houdini19.5/otls/Deadline-Cloud.hda"),
+        (temp_parm, "temp:$OS.rat"),
```

Comment:
Can some additional cases be added or parameterized so that we have `opdef:` `op:` and `oplib:` being skipped captured in the tests.